src/linear_algebra/finite_dimensional.lean

```diff
@@ -793,6 +793,27 @@ section top
 theorem findim_top : findim K (⊤ : submodule K V) = findim K V :=
 by { unfold findim, simp [dim_top] }
 
+lemma algebra_hom.bijective {F : Type*} [field F] {E : Type*} [field E] [algebra F E]
+  [finite_dimensional F E] (ϕ : E →ₐ[F] E) : function.bijective ϕ :=
+begin
+  have inj : function.injective ϕ.to_linear_map := ϕ.to_ring_hom.injective,
+  have rank_nullity := linear_map.findim_range_add_findim_ker ϕ.to_linear_map,
```

Comment:
Do we not have a lemma about injective maps between fd vector spaces of the same dimension being bijective?
If not, then this proof should be factored out into that lemma.

Reply:
It looks like we do, and it's actually right below!